src/cryptography/utils.py

```diff
@@ -16,6 +16,10 @@
 # cycle ends.
 PersistentlyDeprecated = DeprecationWarning
 DeprecatedIn19 = DeprecationWarning
+# Asymmetric signature and verification contexts were deprecated in 2.0
+# However they are widely used and we will obey a longer than normal
+# deprecation cycle for them.
+PersistentlyDeprecatedIn20 = PendingDeprecationWarning
```

Comment:
My main thinking was that it's useful to know at what point we persistently deprecated something. By grouping the persistent deprecations we have to look at the repository history or changelog to know how long it's been. Having written those words I think I'm going to just use `PersistentlyDeprecated` though, because looking at the changelog is not a burden.